lib/consult/template_functions.rb

```diff
@@ -16,7 +16,7 @@ def secrets(path)
     ############
     # Consul
     ############
-    def service(key, scope: :all, options: nil, meta: nil)
+    def service(key, scope: :all, options: {}, meta: {})
```

Comment:
The code was sending nil for options params but is better to send an empty hash, this issue was present:
```
undefined method `[]' for nil:NilClass
custom_params << use_named_parameter('wait', options[:wait]) if options[:wait]
```
the options variable was nil on the following file of the gem 
```
diplomat-2.6.4/lib/diplomat/service.rb
```